crates/bitwarden/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Changed
```

Comment:
Nit: Should this be `Added` instead?

Reply:
Good catch fixed in https://github.com/bitwarden/sdk/pull/678/commits/a47032d45dbab4b7c87e7fd5baa8caf219bc863a